commit 8675fa58c4ed7dfc71302b5c92697cc1e36cab53
Author: Jerome Leclanche <adys.wh@gmail.com>
Date:   Sat Apr 20 17:38:47 2013 +0100

    doc: Fix @itemx references without corresponding @item
    
    This is an error in texinfo>=5.0

diff --git a/doc/msmtp.texi b/doc/msmtp.texi
index a9c8e46..4012271 100644
--- a/doc/msmtp.texi
+++ b/doc/msmtp.texi
@@ -54,7 +54,7 @@ notice are preserved. These files are offered as-is, without any warranty.
 * Authentication::			How to use authentication.
 * Delivery Status Notifications::	How to use DSN.
 * Sendmail mode::			How to send mail.
-* Server information mode::		How to obtain information about 
+* Server information mode::		How to obtain information about
 					an SMTP server.
 * Remote Message Queue Starting mode::	How to send RMQS requests.
 * Examples::				Usage examples.
@@ -64,7 +64,7 @@ notice are preserved. These files are offered as-is, without any warranty.
 @node Introduction
 @chapter Introduction
 
-msmtp is an SMTP client. 
+msmtp is an SMTP client.
 
 In its default mode of operation, it reads a mail from standard input and sends
 it to a predefined SMTP server that takes care of proper delivery. Command line
@@ -112,19 +112,19 @@ and so on.  Each configuration file can define multiple accounts.
 msmtp supports a system wide configuration file and a user configuration file.
 Both are optional and need not exist.
 
-If it exists and is readable, a system wide configuration file 
-@code{SYSCONFDIR/msmtprc} will be loaded, where @code{SYSCONFDIR} depends on 
-your platform. The default is @code{/usr/local/etc}. 
+If it exists and is readable, a system wide configuration file
+@code{SYSCONFDIR/msmtprc} will be loaded, where @code{SYSCONFDIR} depends on
+your platform. The default is @code{/usr/local/etc}.
 Use @code{--version} to find out which directory your version uses.
 
-If it exists and is readable, a user configuration file will be loaded 
+If it exists and is readable, a user configuration file will be loaded
 (@code{~/.msmtprc} by default). Accounts defined in the user configuration
 file override accounts from the system configuration file.  The user
-configuration file must have no more permissions than user read/write.  
+configuration file must have no more permissions than user read/write.
 Configuration data from either file can be changed by command line options.
-					         
-A configuration file is a simple text file. Empty lines and comment lines 
-(whose first non-blank character is '#') are ignored. Every other line must 
+
+A configuration file is a simple text file. Empty lines and comment lines
+(whose first non-blank character is '#') are ignored. Every other line must
 contain a command and may contain an argument to that command.
 The argument may be enclosed in double quotes (").
 
@@ -162,10 +162,10 @@ Every account definition must contain this command.
 @anchor{port}
 @item port @var{number}
 @cmindex port
-The port that the SMTP server listens on. 
+The port that the SMTP server listens on.
 The default port will be acquired from your operating system's service database:
-for SMTP, the service is "smtp" (default port 25), unless TLS 
-without STARTTLS is used, in which case it is "smtps" (465). For LMTP, it is 
+for SMTP, the service is "smtp" (default port 25), unless TLS
+without STARTTLS is used, in which case it is "smtps" (465). For LMTP, it is
 "lmtp".
 @anchor{timeout}
 @item timeout (off|@var{seconds})
@@ -220,7 +220,7 @@ find it. First, if @samp{passwordeval} is set, it will evaluate that command. If
 @samp{passwordeval} is not set, msmtp will try to find the password in
 @code{~/.netrc}. If that fails, it will try to find it in
 @code{SYSCONFDIR/netrc} (use @code{--version} to find out what @code{SYSCONFDIR}
-is on your platform). If that fails, it will try to get it from a system 
+is on your platform). If that fails, it will try to get it from a system
 specific keyring (if available). If that fails but a controlling terminal is
 available, msmtp will prompt you for it.
 @xref{Authentication}.
@@ -232,8 +232,8 @@ execution of @var{eval}.
 @anchor{ntlmdomain}
 @item ntlmdomain [@var{ntlmdomain}]
 @cmindex ntlmdomain
-Set a domain for the @samp{ntlm} authentication method. The default is to use 
-no domain (equivalent to an empty argument), but some servers seem to require 
+Set a domain for the @samp{ntlm} authentication method. The default is to use
+no domain (equivalent to an empty argument), but some servers seem to require
 one, even if it is an arbitrary string.
 @end table
 
@@ -249,13 +249,13 @@ This command enables or disables TLS/SSL encrypted connections to the SMTP
 server. Not every server supports TLS, and a few that support it require the
 @samp{tls_starttls off} command.@*
 To use TLS/SSL, it is required to either use the @samp{tls_trust_file} command
-(highly recommended) or to disable @samp{tls_certcheck}. 
+(highly recommended) or to disable @samp{tls_certcheck}.
 @xref{Transport Layer Security}.
 @anchor{tls_trust_file}
 @item tls_trust_file [@var{file}]
 @cmindex tls_trust_file
 This command activates strict server certificate verification.
-The given file must contain one or more certificates of trusted Certification 
+The given file must contain one or more certificates of trusted Certification
 Authorities (CAs) in PEM format.@*
 On Debian based systems, you can install the @samp{ca-certificates} package and
 use the file @file{/etc/ssl/certs/ca-certificates.crt}.@*
@@ -307,7 +307,7 @@ alias for @samp{tls_certcheck off}.
 @anchor{tls_starttls}
 @item tls_starttls [(on|off)]
 @cmindex tls_starttls
-This command enables or disables the use of the STARTTLS SMTP command to start 
+This command enables or disables the use of the STARTTLS SMTP command to start
 TLS encryption. It is enabled by default.
 @xref{Transport Layer Security}.
 For compatibility with older versions, @samp{tls_nostarttls} is accepted as an
@@ -315,7 +315,7 @@ alias for @samp{tls_starttls off}.
 @anchor{tls_force_sslv3}
 @item tls_force_sslv3 [(on|off)]
 @cmindex tls_force_sslv3
-Force TLS/SSL version SSLv3. This might be needed to use SSL with some old and 
+Force TLS/SSL version SSLv3. This might be needed to use SSL with some old and
 broken servers. Do not use this unless you have to.
 @xref{Transport Layer Security}.
 @anchor{tls_min_dh_prime_bits}
@@ -331,9 +331,9 @@ remote server.
 @item tls_priorities [@var{priorities}]
 @cmindex tls_priorities
 Set the priorities for TLS sessions. The default is set by the TLS library and
-can be selected by using an empty argument to this command.  Currently this 
+can be selected by using an empty argument to this command.  Currently this
 command only works with sufficiently recent GnuTLS releases. See the GnuTLS
-documentation of the @samp{gnutls_priority_init} function for a description of 
+documentation of the @samp{gnutls_priority_init} function for a description of
 the @var{priorities} string.
 @xref{Transport Layer Security}.
 @end table
@@ -351,30 +351,30 @@ Enable or disable automatic envelope-from addresses. The default is
 When enabled, an envelope-from address of the form user@@domain will be
 generated.  The local part will be set to @code{USER} or, if that fails, to
 @code{LOGNAME} or, if that fails, to the login name of the current user.  The
-domain part can be set with the @samp{maildomain} command (see 
+domain part can be set with the @samp{maildomain} command (see
 @ref{maildomain}).  If the maildomain is empty, the envelope-from address will
 only consist of the user name and not have a domain part.@*
-When disabled, the envelope-from address must be set explicitly with the 
+When disabled, the envelope-from address must be set explicitly with the
 @samp{from} command (see @ref{from}).@*
 @xref{Envelope-from address}.
 @anchor{from}
 @item from [@var{address}]
 @cmindex from
-Set the envelope-from address. This address will only be used when 
+Set the envelope-from address. This address will only be used when
 @samp{auto_from} is disabled.
 @xref{Envelope-from address}.
 @anchor{maildomain}
 @item maildomain [@var{domain}]
 @cmindex maildomain
-Set a domain part for the generation of an envelope-from address. This is only 
+Set a domain part for the generation of an envelope-from address. This is only
 used when @samp{auto_from} is enabled. The domain may be empty.
 @xref{Envelope-from address}.
 @anchor{dsn_notify}
 @item dsn_notify (off|@var{condition})
 @cmindex dsn_notify
-This command sets the condition(s) under which the mail system should send DSN 
-(Delivery Status Notification) messages. The argument off disables 
-explicit DSN requests, which means the mail system decides when to send DSN 
+This command sets the condition(s) under which the mail system should send DSN
+(Delivery Status Notification) messages. The argument off disables
+explicit DSN requests, which means the mail system decides when to send DSN
 messages. This is the default.
 The @var{condition} must be @samp{never}, to never request notification, or a
 comma separated list (no spaces!) of one or more of the following:
@@ -397,7 +397,7 @@ The SMTP server must support the DSN extension.
 @anchor{keepbcc}
 @item keepbcc [(on|off)]
 @cmindex keepbcc
-This command controls whether to remove or keep the Bcc header when sending a 
+This command controls whether to remove or keep the Bcc header when sending a
 mail. The default is to remove it. @xref{Bcc header}.
 @anchor{logfile}
 @item logfile [@var{file}]
@@ -409,7 +409,7 @@ information to standard output.
 @anchor{syslog}
 @item syslog [(on|off|@var{facility})]
 @cmindex syslog
-This command enables or disables syslog logging. The facility can be one of 
+This command enables or disables syslog logging. The facility can be one of
 @samp{LOG_USER}, @samp{LOG_MAIL}, @samp{LOG_LOCAL0}, @dots{}, @samp{LOG_LOCAL7}.
 The default facility is @samp{LOG_USER}. Syslog logging is disabled by default.
 @xref{Logging}.
@@ -446,7 +446,7 @@ addresses.  This is the default.
 @item Server information mode:@*
 @code{msmtp [@var{option}@dots{}] --serverinfo}
 @item Remote Message Queue Starting mode:@*
-@code{msmtp [@var{option}@dots{}] 
+@code{msmtp [@var{option}@dots{}]
 --rmqs=(@var{host}|@var{@@domain}|@var{#queue})}
 @end itemize
 
@@ -495,7 +495,7 @@ about supported features (mail size limit, authentication, TLS, DSN, @dots{})
 and about the TLS certificate (if TLS is active).
 @xref{Server information mode}.
 @anchor{--rmqs}
-@itemx --rmqs=(@var{host}|@var{@@domain}|@var{#queue})
+@item --rmqs=(@var{host}|@var{@@domain}|@var{#queue})
 @opindex --rmqs
 Send a Remote Message Queue Starting request for the given host, domain, or
 queue to the SMTP server and exit.
@@ -510,7 +510,7 @@ Please refer to @ref{Configuration files} for detailed information.
 @itemx --file=@var{filename}
 @opindex -C
 @opindex --file
-Use the given file instead of @code{~/.msmtprc} as the user configuration 
+Use the given file instead of @code{~/.msmtprc} as the user configuration
 file.
 @item -a @var{account}
 @itemx --account=@var{account}
@@ -519,82 +519,82 @@ file.
 Use the given account instead of the account named @samp{default}. This option
 cannot be used together with the @samp{--host} option.
 @xref{Choosing an account}.
-@itemx --host=@var{hostname}
+@item --host=@var{hostname}
 @opindex --host
 Use this SMTP server with settings from the command line; do not use any
 configuration file data. This option cannot be used together with the
 @samp{--account} option. It disables loading of configuration files.
 @xref{Choosing an account}.
-@itemx --port=@var{number}
+@item --port=@var{number}
 @opindex --port
 Set the port number to connect to. @xref{port}.
-@itemx --timeout=(off|@var{seconds})
+@item --timeout=(off|@var{seconds})
 @opindex --timeout
 Set a network timeout. @xref{timeout}. For compatibility with older versions,
 @samp{--connect-timeout} is accepted as an alias for this option.
-@itemx --protocol=(smtp|lmtp)
+@item --protocol=(smtp|lmtp)
 @opindex --protocol
 Set the protocol. @xref{protocol}.
-@itemx --domain=[@var{argument}]
+@item --domain=[@var{argument}]
 @opindex --domain
 Set the argument of the SMTP EHLO (or LMTP LHLO) command. @xref{domain}.
 @anchor{--auth}
-@itemx --auth[=(on|off|@var{method})]
+@item --auth[=(on|off|@var{method})]
 @opindex --auth
 Enable or disable authentication and optionally choose the method. @xref{auth}.
 @anchor{--user}
-@itemx --user=[@var{username}]
+@item --user=[@var{username}]
 @opindex --user
 Set or unset the user name for authentication. @xref{user}.
 @anchor{--passwordeval}
-@itemx --passwordeval=[@var{eval}]
+@item --passwordeval=[@var{eval}]
 @opindex --passwordeval
 Evaluate password for authentication. @xref{passwordeval}.
-@itemx --tls[=(on|off)]
+@item --tls[=(on|off)]
 @opindex --tls
 Enable or disable TLS/SSL. @xref{tls}.
 @anchor{--tls-starttls}
-@itemx --tls-starttls[=(on|off)]
+@item --tls-starttls[=(on|off)]
 @opindex --tls-starttls
 Enable or disable STARTTLS for TLS encryption. @xref{tls_starttls}.
 @anchor{--tls-trust-file}
-@itemx --tls-trust-file=[@var{file}]
+@item --tls-trust-file=[@var{file}]
 @opindex --tls-trust-file
 Set or unset a trust file for TLS encryption. @xref{tls_trust_file}.
 @anchor{--tls-crl-file}
-@itemx --tls-crl-file=[@var{file}]
+@item --tls-crl-file=[@var{file}]
 @opindex --tls-crl-file
-Set or unset a certificate revocation list (CRL) file for TLS. 
+Set or unset a certificate revocation list (CRL) file for TLS.
 @xref{tls_crl_file}.
 @anchor{--tls-fingerprint}
-@itemx --tls-fingerprint=[@var{fingerprint}]
+@item --tls-fingerprint=[@var{fingerprint}]
 @opindex --tls-fingerprint
 Set ot unset the fingerprint of a trusted TLS certificate.
 @xref{tls_fingerprint}.
 @anchor{--tls-key-file}
-@itemx --tls-key-file=[@var{file}]
+@item --tls-key-file=[@var{file}]
 @opindex --tls-key-file
 Set or unset a key file for TLS encryption. @xref{tls_key_file}.
 @anchor{--tls-cert-file}
-@itemx --tls-cert-file=[@var{file}]
+@item --tls-cert-file=[@var{file}]
 @opindex --tls-cert-file
 Set or unset a cert file for TLS encryption. @xref{tls_cert_file}.
 @anchor{--tls-certcheck}
-@itemx --tls-certcheck[=(on|off)]
+@item --tls-certcheck[=(on|off)]
 @opindex --tls-certcheck
-Enable or disable server certificate checks for TLS encryption.  
+Enable or disable server certificate checks for TLS encryption.
 @xref{tls_certcheck}.
 @anchor{--tls-force-sslv3}
-@itemx --tls-force-sslv3[=(on|off)]
+@item --tls-force-sslv3[=(on|off)]
 @opindex --tls-force-sslv3
 Force TLS/SSL version SSLv3. @xref{tls_force_sslv3}.
 @anchor{--tls-min-dh-prime-bits}
-@itemx --tls-min-dh-prime-bits=[@var{bits}]
+@item --tls-min-dh-prime-bits=[@var{bits}]
 @opindex --tls-min-dh-prime-bits
 Set or unset minimum bit size of the Diffie-Hellman (DH) prime.
 @xref{tls_min_dh_prime_bits}.
 @anchor{--tls-priorities}
-@itemx --tls-priorities=[@var{priorities}]
+@item --tls-priorities=[@var{priorities}]
 @opindex --tls-priorities
 Set or unset TLS priorities. @xref{tls_priorities}.
 @end table
@@ -602,9 +602,9 @@ Set or unset TLS priorities. @xref{tls_priorities}.
 @subsection Options specific to sendmail mode
 @table @samp
 @anchor{--auto-from}
-@itemx --auto-from[=(on|off)]
+@item --auto-from[=(on|off)]
 @opindex --auto-from
-Enable or disable automatic envelope-from addresses. The default is off. 
+Enable or disable automatic envelope-from addresses. The default is off.
 @xref{auto_from}.
 @anchor{--from}
 @item -f @var{address}
@@ -613,12 +613,12 @@ Enable or disable automatic envelope-from addresses. The default is off.
 @opindex --from
 Set the envelope-from address. It is only used when @samp{auto_from} is off.
 @xref{from}.@*
-If no account was chosen yet (with @samp{--account} or @samp{--host}), this 
+If no account was chosen yet (with @samp{--account} or @samp{--host}), this
 option will choose the first account that has the given envelope-from address
-(set with the @samp{from} command). If no such account is found, "default" is 
+(set with the @samp{from} command). If no such account is found, "default" is
 used. @xref{Choosing an account}.
 @anchor{--maildomain}
-@itemx --maildomain=[@var{domain}]
+@item --maildomain=[@var{domain}]
 @opindex --maildomain
 Set the domain part for generated envelope-from addresses. It is only used when
 @samp{auto_from} is on. @xref{maildomain}.
@@ -634,10 +634,10 @@ Set or unset DSN notification conditions. @xref{dsn_notify}.
 @opindex -R
 @opindex --dsn-return
 Set or unset the DSN notification amount. @xref{dsn_return}.
-Note that @samp{hdrs} is accepted as an alias for @samp{headers} to be 
+Note that @samp{hdrs} is accepted as an alias for @samp{headers} to be
 compatible with sendmail.
 @anchor{--keepbcc}
-@itemx --keepbcc[=(on|off)]
+@item --keepbcc[=(on|off)]
 @opindex --keepbcc
 Enable or disable the preservation of the Bcc header. @xref{keepbcc}.
 @anchor{--logfile}
@@ -647,7 +647,7 @@ Enable or disable the preservation of the Bcc header. @xref{keepbcc}.
 @opindex --logfile
 Set or unset the log file. @xref{logfile}.
 @anchor{--syslog}
-@itemx --syslog[=(on|off|@var{facility})]
+@item --syslog[=(on|off|@var{facility})]
 @opindex --syslog
 Enable or disable syslog logging. @xref{syslog}.
 @item -t
@@ -659,22 +659,22 @@ mail in addition to the recipients given on the command line.@*
 If any Resent- headers are present, then the addresses from any Resent-To,
 Resent-Cc, and Resent-Bcc headers in the first block of Resent- headers are
 used instead.
-@itemx --read-envelope-from
+@item --read-envelope-from
 @opindex --read-envelope-from
 Read the envelope from address from the From header of the mail.
 Currently this header must be on a single line for this option to work
 correctly.
-@itemx --aliases=[@var{file}]
+@item --aliases=[@var{file}]
 @opindex --aliases
 Set or unset an aliases file. @xref{aliases}.
-@itemx --
+@item --
 This marks the end of options. All following arguments will be treated as
 recipient addresses, even if they start with a '-'.
 @end table
 
 The following options are accepted but ignored for sendmail compatibility:
 @samp{-B@var{type}}, @samp{-bm}, @samp{-F@var{name}}, @samp{-G}, @samp{-hN},
-@samp{-i}, @samp{-L @var{tag}}, @samp{-m}, @samp{-n}, 
+@samp{-i}, @samp{-L @var{tag}}, @samp{-m}, @samp{-n},
 @samp{-O option=@var{value}}, @samp{-ox @var{value}}
 
 @anchor{Choosing an account}
@@ -685,7 +685,7 @@ It depends on the circumstances which method is the best.
 
 @enumerate
 @item @samp{--account=@var{account}}@*
-Use the given account. Command line settings override configuration file 
+Use the given account. Command line settings override configuration file
 settings.
 @item @samp{--host=@var{hostname}}@*
 Use only the settings from the command line; do not use any configuration file
@@ -712,8 +712,8 @@ The system configuration file. Use the @samp{--version} option to find out what
 @item @code{~/.msmtprc}
 The default user configuration file.
 @item @code{~/.netrc} and @code{SYSCONFDIR/netrc}
-The @code{netrc} file contains login information. If a password is not found 
-in the configuration file, msmtp will search it in @code{~/.netrc} and 
+The @code{netrc} file contains login information. If a password is not found
+in the configuration file, msmtp will search it in @code{~/.netrc} and
 @code{SYSCONFDIR} before prompting the user for it. The syntax of @code{netrc}
 files is described in the @code{netrc(5)} or @code{ftp(1)} manual page.
 @end table
@@ -723,12 +723,12 @@ files is described in the @code{netrc(5)} or @code{ftp(1)} manual page.
 
 @table @samp
 @item @code{USER}, @code{LOGNAME}
-These variables override the user's login name when constructing an 
+These variables override the user's login name when constructing an
 envelope-from address. @code{LOGNAME} is only used if @code{USER} is unset.
 @item @code{TMPDIR}
-Directory to create temporary files in. If this is unset, a system specific 
+Directory to create temporary files in. If this is unset, a system specific
 default directory is used.@*
-A temporary file is only created when the @samp{-t}/@samp{--read-recipients} 
+A temporary file is only created when the @samp{-t}/@samp{--read-recipients}
 or @samp{--read-envelope-from} option is used. The file is then used to buffer
 the headers of the mail (but not the body, so the file won't get very large).
 @item @code{EMAIL}, @code{SMTPSERVER}
@@ -761,14 +761,14 @@ port for this mode is 465 (smtps).
 client sends the STARTTLS command when it wishes to begin TLS encryption. The
 default port for this mode is the default SMTP port: 25 (smtp).
 @end itemize
-msmtp can switch between these modes with the @samp{tls_starttls} command (see 
-@ref{tls_starttls}) command or the @samp{--tls-starttls} option (see 
+msmtp can switch between these modes with the @samp{tls_starttls} command (see
+@ref{tls_starttls}) command or the @samp{--tls-starttls} option (see
 @ref{--tls-starttls}).
 
 When TLS is started, the server sends a certificate to identify itself. This
 certificate contains information about the certificate owner, the certificate
 issuer, and the activation and expiration times of the certificate.  This
-information can be displayed in server information mode. 
+information can be displayed in server information mode.
 @xref{Server information mode}.
 
 To use TLS, it is required to either enable full server certificate verification
@@ -821,20 +821,20 @@ and @ref{tls_priorities} commands.
 @chapter Authentication
 
 Many SMTP servers require a client to authenticate itself before it is allowed
-to send mail. 
+to send mail.
 
 Multiple authentication methods exist. Most SMTP servers support only some of
 them.  Some methods send authentication data in plain text (or nearly plain
 text) to the server. These methods should only be used when TLS is active to
 prevent others from stealing the password. @xref{Transport Layer Security}.
 
-By default, msmtp chooses a method automatically, and it will never choose one 
+By default, msmtp chooses a method automatically, and it will never choose one
 that puts the authentication data at risk. See below for details.
 
 msmtp supports the following authentication methods:
 @itemize
 @item @samp{PLAIN}@*
-This authentication method needs a user name and a password. 
+This authentication method needs a user name and a password.
 Both are send in BASE64 encoding, which can be easily decoded to plain text.
 @item @samp{SCRAM-SHA-1}@*
 This authentication method needs a user name and a password.
@@ -867,11 +867,11 @@ The authentication data is not sent in plain text, but the encryption based on
 MD5 is not considered secure anymore.
 @item @samp{LOGIN}@*
 This is a non-standard authentication method similar to (but worse than) PLAIN.
-It needs a user name and a password, both of which are send in BASE64 encoding, 
+It needs a user name and a password, both of which are send in BASE64 encoding,
 which can be easily decoded to plain text.
 @item @samp{NTLM}@*
 This is an obscure non-standard authentication method. It needs a user name and
-a password and in some cases a special domain parameter (see @ref{ntlmdomain}). 
+a password and in some cases a special domain parameter (see @ref{ntlmdomain}).
 The authentication data is not send in plain text, but since NTLM is not an open
 standard, it should be considered broken and insecure.
 @end itemize
@@ -887,7 +887,7 @@ find it. First, if @samp{passwordeval} is set, it will evaluate that command. If
 @samp{passwordeval} is not set, msmtp will try to find the password in
 @code{~/.netrc}. If that fails, it will try to find it in
 @code{SYSCONFDIR/netrc} (use @code{--version} to find out what @code{SYSCONFDIR}
-is on your platform). If that fails, it will try to get it from a system 
+is on your platform). If that fails, it will try to get it from a system
 specific keyring (if available). If that fails but a controlling terminal is
 available, msmtp will prompt you for it.
 
@@ -898,8 +898,8 @@ Access GUI application. The @samp{account name} is same as the msmtp @samp{user}
 argument. The @samp{keychain item name} is @code{smtp://<hostname>} where
 @code{<hostname>} matches the msmtp @samp{host} argument.
 
-The authentication method can be chosen with the @samp{auth} command or 
-@samp{--auth} option, but it is usually sufficient to just use the @samp{on} 
+The authentication method can be chosen with the @samp{auth} command or
+@samp{--auth} option, but it is usually sufficient to just use the @samp{on}
 argument to let msmtp choose the method itself. See @ref{auth}, @ref{--auth}.
 
 If msmtp chooses the method itself, it will never choose an insecure method.
@@ -915,8 +915,8 @@ do that by manually setting the authentication method while TLS is off.
 @node Delivery Status Notifications
 @chapter Delivery Status Notifications
 
-In situations such as delivery failure or very long delivery delay, the mail 
-system often generates a message for the sender of the mail in question, 
+In situations such as delivery failure or very long delivery delay, the mail
+system often generates a message for the sender of the mail in question,
 informing him about the difficulties.
 
 Delivery Status Notification (DSN) requests, defined in RFC 3461, try to give
@@ -925,10 +925,10 @@ The SMTP server must support the DSN extension. @xref{Server information mode}.
 
 A first parameter controls when such messages should be generated:
 never, on delivery failure, on delivery delay, and/or on success.
-This can be set with @samp{dsn_notify}/@samp{--dsn-notify}, see 
+This can be set with @samp{dsn_notify}/@samp{--dsn-notify}, see
 @ref{dsn_notify}/@ref{--dsn-notify}.
 
-A second parameter controls how much of the original mail should be contained 
+A second parameter controls how much of the original mail should be contained
 in a DSN message: only the headers, or the full mail. This can be set with
 @samp{dsn_return}/@samp{--dsn-return}, see @ref{dsn_return}/@ref{--dsn-return}.
 Note that this parameter only applies to DSNs that indicate delivery failure
@@ -962,8 +962,8 @@ When @samp{auto_from} is enabled, an envelope-from address of the form
 user@@domain will be generated.  The local part will be set to @code{USER} or,
 if that fails, to @code{LOGNAME} or, if that fails, to the login name of the
 current user.  The domain part can be set with the @samp{maildomain} command
-and @samp{--maildomain} option (see @ref{maildomain}).  If the maildomain is 
-empty, the envelope-from address will only consist of the user name and not 
+and @samp{--maildomain} option (see @ref{maildomain}).  If the maildomain is
+empty, the envelope-from address will only consist of the user name and not
 have a domain part.
 
 @node Logging
@@ -972,33 +972,33 @@ have a domain part.
 Logging is enabled on a per account basis. If it is enabled, msmtp will generate
 one log line for each mail it tries to send via the account in question.
 
-The line will include the following information: 
+The line will include the following information:
 @itemize
 @item Host name of the SMTP server:  @code{host=hostname}
 @item Whether TLS was used: @code{tls=(on|off)}
 @item Whether authentication was used: @code{auth=(on|off)}
-@item The user name used for authentication (only if authentication is used): 
+@item The user name used for authentication (only if authentication is used):
 @code{user=name}
 @item The envelope-from address: @code{from=address}
 @item The recipient addresses: @code{recipients=addr1,addr2,@dots{}}
 @item The size of the mail as transferred to the server, in bytes (only if the
 delivery succeeded): @code{mailsize=number}
 @item The SMTP status code and SMTP error message (only in case of
-failure and only if available): @code{smtpstatus=number}, 
+failure and only if available): @code{smtpstatus=number},
 @code{smtpmsg='message'}. Multiline SMTP messages will be concatenated into one
 line.
 @item The msmtp error message (only in case of failure and only
 if available): @code{errormsg='message'}
-@item The msmtp exit code (from @code{sysexits.h}; @samp{EX_OK} indicates 
+@item The msmtp exit code (from @code{sysexits.h}; @samp{EX_OK} indicates
 success): @code{exitcode=EX_@dots{}}
 @end itemize
 
-If a logfile is given with the @samp{logfile} command or @samp{--logfile} 
-option, this log line will be prepended with the current date and time and 
+If a logfile is given with the @samp{logfile} command or @samp{--logfile}
+option, this log line will be prepended with the current date and time and
 appended to the specified file. See @ref{logfile}, @ref{--logfile}.
 
 If syslog logging is enabled with the @samp{syslog} command or @samp{--syslog}
-option, the log line is passed to the syslog service with the specified 
+option, the log line is passed to the syslog service with the specified
 facility. See @ref{syslog}, @ref{--syslog}.
 
 @node Bcc header
@@ -1013,7 +1013,7 @@ with the @samp{keepbcc} command and @samp{--keepbcc} option, see
 @node Server information mode
 @chapter Server information mode
 
-In server information mode, msmtp prints as much information about the SMTP 
+In server information mode, msmtp prints as much information about the SMTP
 server as it can get and then exits.
 
 The SMTP features that can be detected are:
@@ -1047,7 +1047,7 @@ be printed about the SMTP server's TLS certificate (if available):
 @item Country
 @end itemize
 @item Issuer information
-@itemize 
+@itemize
 @item Common Name
 @item Organization
 @item Organizational unit
@@ -1056,7 +1056,7 @@ be printed about the SMTP server's TLS certificate (if available):
 @item Country
 @end itemize
 @item General
-@itemize 
+@itemize
 @item Activation time
 @item Expiration time
 @item SHA1 fingerprint
@@ -1147,7 +1147,7 @@ password secret
 account freemail2 : freemail
 from joey@@freemail.example
 
-# The SMTP server of the provider. 
+# The SMTP server of the provider.
 account provider
 host mail.provider.example
 from smithjoe@@provider.example
@@ -1178,7 +1178,7 @@ msmtp.  Therefore msmtp chooses the first account that matches the from address
 you@@example.com. Alternatively, you can use the @option{-a} option:
 @example
 set sendmail="/path/to/msmtp -a my_account"
-@end example 
+@end example
 
 Or set everything from the command line:
 @example
@@ -1196,9 +1196,9 @@ macro generic "<esc>3" ":set from=you@@some-other-provider.example"
 @end example
 Now you can use <esc>1, <esc>2, and <esc>3 to switch accounts.
 
-The following example uses a different approach: it maps the single key 
-@code{<tab>} in Compose context for switching between the various account in a 
-handy visual way. In the same Compose context, @code{=} is mapped in order to 
+The following example uses a different approach: it maps the single key
+@code{<tab>} in Compose context for switching between the various account in a
+handy visual way. In the same Compose context, @code{=} is mapped in order to
 show the current msmtp account. This example was contributed by Thomas Baruchel.
 @example
 # Define <tab> and = in order to switch or see the current msmtp account
